.travis.yml

```diff
@@ -41,7 +41,7 @@ before_install:
   - export PATH=$HOME/.yarn/bin:$PATH
 
 install:
-  - yarn install --no-lockfile --non-interactive
```

Comment:
Yarn lockfiles are not respected by consuming apps, so if we want to use a lockfile for development (which I'm in favor of), we should still let CI ignore the lockfile as this will more closely reflect the sub-dependencies that a consuming app would get when it installs the addon.

See the recommended addon blueprint here: https://github.com/ember-cli/ember-cli/blob/6f2d49a341399d255b40e2bd832b31b7a9bdf8c3/blueprints/addon/files/.travis.yml#L68

Reply:
Good point